.github/workflows/mybinder-cache.yaml

```diff
@@ -0,0 +1,12 @@
+name: Binder
+on: [push]
+
+jobs:
+  Create-MyBinderOrg-Cache:
+    runs-on: ubuntu-latest
+    steps:
+    - name: cache binder build on mybinder.org
+      uses: jupyterhub/repo2docker-action@master
+      with:
+        NO_PUSH: true
+        MYBINDERORG_TAG: ${{ github.event.ref }} # This builds the container on mybinder.org with the branch that was pushed on.
```

Comment:
What branch does mybinder show by default? Do all the available branches appear in a selection panel like readthedocs? I ask because if mybinder only shows the latest build by default than we probably want this action to only run on commits to the master branch. We wouldn't want PRs to automatically trigger updates to the environment.

By the way, this doesn't seem to work, when I follow the link you posted in the README, it starts building and fails with the error: "Jupyterlab failed to build"

Reply:
We can set this to only fire on commits to master, but by default mybinder tags the container with the hash of the commit and the branch is selected by the url (or you can also set the commit in the url as well). We don't want to thrash their infrastructure by having a bunch of builds, but one advantage of building images on PRs is that then mybinder can be used to test. 

I think we should start with only commits to master, I'll make that change

Reply:
ok